package.json

```diff
@@ -12,6 +12,7 @@
     "packages/*"
   ],
   "scripts": {
+    "install": "lerna bootstrap --hoist",
```

Comment:
But why `"test": "lerna run test"` is ok.

Reply:
> But why `"test": "lerna run test"` is ok.

`test` is expected to be executed after `install/bootstrap` because `test` relies on test libraries. So `lerna` has been added on running `test` and `npx` could be omitted.

Reply:
> > But why `"test": "lerna run test"` is ok.
> 
> `test` is expected to be executed after `install/bootstrap` because `test` relies on test libraries. So `lerna` has been added on running `test` and `npx` could be omitted.

I see.